src/toxcore/onion/client/mod.rs

```diff
@@ -127,6 +129,14 @@ struct OnionFriend {
     search_count: u32,
     /// Time when this friend was seen online last time
     last_seen: Option<Instant>,
+    /// Number of files sending.
+    pub num_sending_files: u8,
+    /// File transfer objects for sending.
+    pub files_sending: Arc<RwLock<Vec<Option<FileTransfers>>>>,
```

Comment:
In the future we can get rid of Onion, so there should be messenger/friend with these fields like num_sending_files, files_sending, files_receiving, status etc, and with a field with a link to OnionFriend

Reply:
I think this would be in other PR.

Reply:
OK